nssp/tests/test_patch.py

```diff
@@ -0,0 +1,179 @@
+import pytest
+from unittest.mock import patch as mock_patch, call
+from delphi_nssp.patch import patch, good_patch_config
+import os
+import shutil
+from datetime import datetime, timedelta
+import pandas as pd
+
+class TestPatchModule:
+    @mock_patch('logging.Logger')
+    def test_good_patch_config_case1(self, mock_logger):
+        # Case 1: missing custom_run flag and patch section
+        patch_config = {
+            "common": {}
+        }
+        assert not good_patch_config(patch_config, mock_logger)
+        mock_logger.error.assert_has_calls([
+            call("Calling patch.py without custom_run flag set true."),
+            call("Custom flag is on, but patch section is missing."),
+        ])
+
+    @mock_patch('logging.Logger')
+    def test_good_patch_config_case2(self, mock_logger):
```

Comment:
can you renameing these to describe the the behavior like test_bad_config_missing_end

Reply:
Added. [6c0abad](https://github.com/cmu-delphi/covidcast-indicators/pull/2000/commits/6c0abad7af8b01d79aded2fec06864ba67a9f7e0)